packages/redux-query-react/index.d.ts

```diff
@@ -45,20 +48,17 @@ declare module 'redux-query-react' {
   }
   export type ReduxQueryProvider = React.ComponentType<ProviderProps>;
 
-  export type ActionPromise<TEntities = Entities> =
-    | Promise<ActionPromiseValue<TEntities>>
-    | undefined;
   type RunMutation<TEntities = Entities, TArgs = never> = (
     ...args: TArgs
-  ) => ActionPromise<TEntities>;
+  ) => Promise<ActionPromiseValue<TEntities>>;
```

Comment:
should match: https://github.com/amplitude/redux-query/blob/master/packages/redux-query-react/src/hooks/use-mutation.js#L13-L15